source/NVDAObjects/__init__.py

```diff
@@ -578,12 +602,36 @@ def _get_rowCount(self):
 		"""
 		raise NotImplementedError
 
+	def _get_presentationalRowCount(self):
+		"""
+		An optional version of the rowCount property used purely for speech and braille presentation if implemented.
+		This is never used for navigational logic.
+		This property should be implemented if the table has virtual content which may not all be loaded at one time.
+		For example, a table with 1000 rows and 1000 columns, yet the table only shows perhaps 10 rows by 10 columns at a time.
+		Although the  rowCount might be 10, the user needs to  be told the table really has 1000 rows. 
+		If the underlying APIs do not distinguish between virtual and physical cell coordinates, then this property should not be implemented.
+		@rtype: int
+		"""
+		raise NotImplementedError
+
 	def _get_columnCount(self):
 		"""Retreaves the number of columns this object contains if its a table.
 		@rtype: int
 		"""
 		raise NotImplementedError
 
+	def _get_presentationalColumnCount(self):
+		"""
+		An optional version of the columnCount property used purely for speech and braille presentation if implemented.
+		This is never used for navigational logic.
+		This property should be implemented if the table has virtual content which may not all be loaded at one time.
+		For example, a table with 1000 rows and 1000 columns, yet the table only shows perhaps 10 rows by 10 columns at a time.
+		Although the  columnCount might be 10, the user needs to  be told the table really has 1000 columns. 
+		If the underlying APIs do not distinguish between virtual and physical cell coordinates, then this property should not be implemented.
+		@rtype: int
+		"""
+		raise NotImplementedError
+
 	def _get_rowSpan(self):

```

Comment:
Just to make sure, could there also be something like a presentational row/column span?

Reply:
I'm not aware of any colspan or rowspan ARIA attributes at this stage. Only rowIndex and rowCount. Spanning would also have an affect on the physical table thus I don't think that could ever be virtualized.